packages/slice-machine/server/src/api/state.ts

```diff
@@ -104,6 +106,26 @@ export default async function handler() {
     env
   );
 
+  // Refresh auth
+  if (!isFake && env.prismicData.auth) {
+    try {
+      const newTokenResponse: Response = await DefaultClient.refreshToken(
```

Comment:
I think we should have a function in the core to refresh the token and change the cookies, like the login or signup does.

Because i'm guessing this logic won't only be used here.